tests/test_time_machine.py

```diff
@@ -434,6 +434,36 @@ def test_destination_datetime_tzinfo_zoneinfo_no_orig_tz():
         with time_machine.travel(dest):
             assert time.tzname == ("EAT", "EAT")
 
+        dest2 = LIBRARY_EPOCH_DATETIME.replace(tzinfo=ZoneInfo("UTC"))
```

Comment:
Happy to split this into a new test named `test_destination_datetime_tzinfo_zoneinfo_utc_no_orig_tz` if this makes things clearer.

Reply:
One test per variant please! Tests should be as small as possible, so they can pass or fail independently.

Reply:
That's done :)